Content.Server/Temperature/Systems/TemperatureDamageSystem.Containers.cs

```diff
@@ -0,0 +1,122 @@
+using Content.Server.Temperature.Components;
+
+namespace Content.Server.Temperature.Systems;
+
+public sealed partial class TemperatureDamageSystem
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="uid"></param>
+    /// <param name="component"></param>
+    /// <param name="args"></param>
+    private void OnParentChange(Entity<TemperatureDamageThresholdsComponent> entity, ref EntParentChangedMessage args)
```

Comment:
Either write proper documentation or don't use xmldoc at all. It's not required for private functions, and the params aren't even correct here, there's no `uid` or `component` param.

(fix this on **all functions** not just this one)